tests/container-kill_test.go

```diff
@@ -0,0 +1,140 @@
+package exp
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	chaosClient "github.com/litmuschaos/chaos-operator/pkg/client/clientset/versioned/typed/litmuschaos/v1alpha1"
+	"github.com/mayadata-io/chaos-ci-lib/pkg/utils"
+	chaosTypes "github.com/mayadata-io/chaos-ci-lib/types"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	scheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog"
+)
+
+var (
+	kubeconfig     string
+	config         *restclient.Config
+	client         *kubernetes.Clientset
+	clientSet      *chaosClient.LitmuschaosV1alpha1Client
+	err            error
+	experimentName = "container-kill"
+	engineName     = "engine2"
+)
+
+func TestChaos(t *testing.T) {
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "BDD test")
+}
+
+var _ = BeforeSuite(func() {
+	var err error
+	kubeconfig = os.Getenv("HOME") + "/.kube/config"
```

Comment:
better to use getConfig() to generate clientset at one place and re use it.

Reply:
will do this